src/rajAJun20/BankSystem.java

```diff
@@ -0,0 +1,66 @@
+package rajAJun20;
+
+public class BankSystem {
+
+	int balance = 50000;
```

Comment:
No Need to initialize this variable. perform credit() method first.

Reply:
Changes done ....!!